manager/src/main/resources/application.properties

```diff
@@ -103,6 +103,7 @@ mp.messaging.connector.smallrye-kafka.health-enabled=false
 
 # Managed Connectors
 managed-connectors.cluster.id=${MC_CLUSTER_ID:fakeClusterId}
+managed-connectors.namespace.id=${MC_NAMESPACE_ID:fakeNamespaceId}
```

Comment:
Is this used somewhere?

Reply:
It will be required once the MC SDK is updated. The introduction of the new `namespace_id` is exactly what triggered all these breaking changes.

Reply:
@jomarko it will be used [here](https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/pull/577/files#diff-9960ccbd853193b5a5ecf650ebcff22e7070995f6cf6ed3ff63905162527a1edR39).